pkgs/development/python-modules/python-baseconv/default.nix

```
@@ -0,0 +1,20 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+}:
+buildPythonPackage rec {
+  pname = "python-baseconv";
+  version = "1.2.2";
+
+  src = fetchPypi {
+    inherit pname version ;
+    sha256 = "0539f8bd0464013b05ad62e0a1673f0ac9086c76b43ebf9f833053527cd9931b";
+  };
+
```

since there's not tests
```suggestion
  pythonImportsCheck = [ "baseconv" ];
  
```

---

Done.